Dockerfile.concourse

```diff
@@ -1,4 +1,4 @@
-FROM onsdigital/dp-concourse-tools-ubuntu
+FROM onsdigital/dp-concourse-tools-ubuntu:ubuntu20.4-rc.1
```

Comment:
Majority of the other repo uses `ubuntu-20:ubuntu20.4-rc.1`

Is this much of a difference?

Reply:
done